packages/application-shell/src/apollo-links/header-link.spec.js

```diff
@@ -51,13 +51,27 @@ describe('with valid target', () => {
     await waitFor(
       execute(link, {
         query,
-        variables: { target: GRAPHQL_TARGETS.MERCHANT_CENTER_BACKEND },
+        variables: {
+          target: GRAPHQL_TARGETS.MERCHANT_CENTER_BACKEND,
+          featureFlag: 'test-feature-a',
+        },
       })
     );
   });
 
   it('should set headers matching snapshot', () => {
-    expect(context).toMatchSnapshot();
+    expect(context).toMatchInlineSnapshot(`
```

Comment:
I find inline snapshots convienent for this when not too long to show what's expected.